king_grm.R

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env Rscript
+library(argparser)
+library(magrittr)
+
+nt_help <- "num.thread argument for snpgdsIBDKING (if NA, detect the number" %>%
+  paste("of cores automatically)")
+argp <- arg_parser("Run KING-robust") %>%
+  add_argument("gds_file", help = "GDS file") %>%
+  add_argument("--out_prefix", help = "Prefix for output files",
+               default = "") %>%
+  add_argument("--variant_id", help = "File with vector of variant IDs") %>%
+  add_argument("--sample_id", help = "File with vector of sample IDs") %>%
+  add_argument("--num_thread", help = nt_help)
```

Comment:
if is NA, `snpgdsIBDKING()` will automatically detect number of available cores

Reply:
In some other scripts I called this argument `num_cores`. Either name is fine but we should be consistent.